crates/astria-composer/Cargo.toml

```diff
@@ -4,6 +4,11 @@ version = "0.3.1"
 edition = "2021"
 rust-version = "1.70.0"
 
+[[bin]]
+name = "astria-composer"
+test = false
```

Comment:
why is this false? doesn't this turn tests off for this crate when running `cargo test`? same for others
https://doc.rust-lang.org/cargo/reference/cargo-targets.html#the-test-field